app/components/Home.vue

```diff
@@ -42,6 +43,10 @@ import { SET_CURRENT_TAB, FIREBASE_INIT } from '../store/constants'
 export default {
   data: () => {
     return {
+      inactiveColor: config.colors.bottomNavigationBar.inactiveColor,
```

Comment:
We could acquire some code reading quality if we use object destructuring to the config colors attributes.